frontend/src/components/IstioWizards/ServiceWizard.tsx

```diff
@@ -725,6 +735,10 @@ class ServiceWizard extends React.Component<ServiceWizardProps, ServiceWizardSta
               this.onConfirmPreview(items);
             }}
           />
+          {!this.props.istioAPIEnabled &&
+            (<div style={{padding: " 0 0 20px 0"}}>
+              <KialiIcon.Warning /> <b>Istio API is disabled.</b> Be careful when editing the configuration as changes may not be compliant (The validations are disabled)
```

Comment:
```suggestion
              <KialiIcon.Warning /> <b>Istio API is disabled.</b> Be careful when editing the configuration as the Istio config validations are disabled when the Istio API is disabled.
```

Reply:
Missed this in my last review but not a blocker to make it consistent with the other messages.